techmatters/flex-plugins
# Connect Contact to Case Banners

## Description
Case Merging Banners:
- Banner for when a contact is added to a case
- Banner for when the contact is removed from a case

TODO:
- Unit tests
- Translations
- Banner inside the case itself (when creating a new case)
- Double check feature flags


### Checklist

### Related Issues
Fixes https://tech-matters.atlassian.net/browse/CHI-1861

### Verification steps
Connect a contact to a case, and check the banner appears. Interact with the banner(s).
